src/main/java/edu/harvard/iq/dataverse/dataaccess/FileAccessIO.java

```diff
@@ -509,6 +509,7 @@ public FileInputStream openLocalFileAsInputStream () {
 
         try {
             in = new FileInputStream(getFileSystemPath().toFile());
+            in.skip(this.getOffset());
```

Comment:
I believe this is what we want to rearrange: this in.skip() should not be happening here, in the open method, but in the setOffset() method itself. Because we want to be able to change that offset after the initial open. 

The setOffset() method will need to throw an IOException, if it's called while the InputStream is still null; or if the skip() call itself results in an IOException()

Reply:
Yes, excellent idea. Implemented in 4724e11 and now offset seems to be working again, at least for non-tabular files. Thanks!